lib/internal/readline/interface.js

```diff
@@ -415,6 +416,11 @@ class Interface extends InterfaceConstructor {
     }
   }
 
+  setLine(line) {
+    this.line = line;
+    this.isMultiline = StringPrototypeIndexOf(line, '\n') !== -1;
```

Comment:
Adding `isMultiline` as property on the REPL exposes this to users and it becomes an official API that way. I don't think we should do that for now. Can you please use an internal symbol instead?